lib/install/opsuser/opsuser.go

```diff
@@ -217,6 +217,10 @@ func (mgr *RBACManager) collectDatastores(ctx context.Context, finder *find.Find
 	}
 	volumeLocations := make([]url.URL, 0, len(mgr.configSpec.Storage.VolumeLocations))
 	for _, volumeLocation := range mgr.configSpec.Storage.VolumeLocations {
+		// Only apply changes to datastores managed by vSphere
+		if volumeLocation.Scheme != "ds" {
```

Comment:
We should move the "ds" scheme into govmomi as a constant and transition vic as a whole to using the govmomi DatastorePath structure.
Not needed for this PR however.